### ISSUE TITLE
Missing container dependencies

### ISSUE THREAD

**Issue by dchaley** [dchaley/qupath-project-initializer#30]
The container is missing (at least) these dependencies:

```
Loading library prism_es2 from resource failed: java.lang.UnsatisfiedLinkError: /root/.openjfx/cache/21+31/amd64/libprism_es2.so: libX11.so.6: cannot open shared object file: No such file or directory
2024-08-14 10:47:32.626 PDT
java.lang.UnsatisfiedLinkError: /root/.openjfx/cache/21+31/amd64/libprism_es2.so: libX11.so.6: cannot open shared object file: No such file or directory
```

```
java.io.IOException: Cannot run program "gcloud": error=2, No such file or directory
```

**Comment by dchaley** [comment]
First [build attempt](https://console.cloud.google.com/cloud-build/builds;region=us-central1/74ffa18b-f365-4e52-a5bb-8e920322742a;step=1?project=deepcell-on-batch) failed: gpg isn't installed in the base image.

```
/bin/sh: 1: gpg: not found
```

**Comment by dchaley** [comment]
Fixed by #32   🎉 

We needed to fetch the `apt` repo contents before attempting to install gpg.
